GameDefinition/Content/ModeKind.hs

```diff
@@ -111,10 +111,10 @@ brawl = ModeKind  -- sparse melee in daylight, with shade for melee ambush
   , mendMsg = [ (Killed, "The inquisitive scholars turned out to be envious of our deep insight to the point of outright violence. It would still not result in such a defeat and recanting of our thesis if we figured out to use terrain to protect us from missiles or even completely hide our presence. It would also help if we honourably kept our ground together to the end, at the same time preventing the overwhelming enemy forces from brutishly ganging up on our modest-sized, though valiant, research team.")
               , (Conquer, "That's settled: local compactness *is* necessary for relative completeness, given the assumptions.") ]
   , mrules  = T.unlines $
-      [ "* one level only"
-      , "* 3 heroes, 3 human foes"
-      , "* avoid losses"
-      , "* incapacitate all foes ASAP"
+      [ "* One level only"
+      , "* Three heroes vs. Three human enemies"
```

Comment:
I also thought the numbers being capitalized looked better but wasn't sure so I wanted to only capitalize the first letter of each line. After looking shortly I found out that it is maybe grammatically correct to capitalize after vs. which would make it look more pleasant in our opinion as well. I will change it to use that convention. In the case of spawned, it would for example look like:
Three heroes vs. Spawned enemies
Which looks fine in my opinion